docs/development/contributing.rst

```diff
@@ -84,4 +84,24 @@ a translation. To add them, do run ``makemessages`` from the ``data/public/stati
    cd data/public/static/
    python ../../../manage.py makemessages --all -d djangojs --ignore jsi18n --ignore admin --ignore CACHE --ignore recurrence --ignore select2
 
-We tend to edit our .po files using weblate, but a local editor like poedit works as well.
\ No newline at end of file
+We tend to edit our .po files using weblate, but a local editor like poedit works as well.
+
+Using other databases
+---------------------
+
+By default, ephios uses a sqlite database for development and PostgreSQL for production.
+
+MySQL
+''''''
+
+We do not recommend using MySQL, but if you need a mysql for development, you can use the following docker command:
```

Comment:
why do we explain how to run a MySQL (which we do not recommend) but do not explain how postgres works?

Reply:
I had to test mysql and remembered to dump it into the docs. I agree we should add postgres as well.